server/@types/activitiesAPI/index.d.ts

```diff
@@ -1922,6 +1922,11 @@ export interface components {
       minimumIncentiveLevel: string
       /** @description The list of minimum education levels that can apply to this activity */
       minimumEducationLevel: components['schemas']['ActivityMinimumEducationLevel'][]
+      /**
+       * @description Whether the activity is live or archived
+       * @example live
+       */
+      activityState: string
```

Comment:
What are the activity state values? I should probably review the API change first, shouldn't I?!

Reply:
Yes :-)